infinity-keys/ik-landing
# tawnee/eng 710 update mobile header

- KABOOM
- Hello html
- add redwood
- Home page
- Start of redwood on render
- yml -> yaml
- Redwood api config
- Yarn install before yarn deploy
- Run setup command
- Enable prerendering test
- Tailwind
- Redwood upgrade
- Testing if migrations are applied to render
- ENG-417: add-global-css-and-fonts (#415)
- ENG-418: add-ui-components (#416)
- ENG-421: add wrapper and related components (#418)
- conflict fix
- empty migration
- ENG-419: get-homepage-working (#419)
- ENG-420: markdown and alert components (#423)
- ENG-424: cloudinary-components (#424)
- ENG-427: email compononets (#429)
- ENG-428: grid components (#430)
- ENG-432: wallet components (#431)
- ENG-433: privacy and 404 pages (#432)
- ENG-434: cookie consent 🍪 (#434)
- ENG-439: fix linter errors (#437)
- ENG-436: skip link (#436)
- ENG-438: create user model in db (#440)
- Rename veercel.json to vercel.json (#491)
- ENG-443: profile cell and page (#448)
- ENG-445: add loading icon functionality (#446)
- Dockerize postgres for easy local db setup (#490)
- Bump postgres to 15
- Return to original db name, but as postgres 15 this time
- ENG-475: trigger reply email (#482)
- ENG-476: add delete info view and logic (#487)
- Update schema for many to many relationships (#499)
- Schema updates (#504)
- itsmekori/eng-492-update-partner-section (#500)
- update hero section with styling and success component
- Update redwood deps (#512)
- ENG-416: migrate rest routes (#520)
- ENG-516: organize sdl files (#528)
- Itsmekori/eng 506 bring over puzzle component UI elements (#519)
- Schema update for rewardable concept (#518)
- ENG-517: migrate nft check api routes (#533)
- Tawnee/eng 526 lens share package (#543)
- Tawnee/eng 527 move utils into packages (#546)
- ENG-525: new puzzles + seed (#544)
- render config api (#555)
- feature/route order (#556)
- add reddit svg to footer (#602)
- ENG-584: hero particles (#587)
- ENG-532: replace web and api libs with packages (#549)
- add alpha holders data query (#603)
- ENG-596: update vite in packages (#604)
- migrate files and renamed component
- update syntax  and fix prettier errors
- update reddit footer link (#607)
- delete index.js file and update padding on instructions
- Eng-547: add step UI (#558)
- ENG-548 Puzzle UI (#568)
- ham/eng 591 standardizing and simplifying api routes (#600)
- Itsmekori/eng 625 create discord mint tracking bot (#622)
- push console logs (#627)
- Discord bot debug (#628)
- update env variable names (#629)
- check if tx confirmed before post (#630)
- migrate custom css, images and update code to show css
- set up styles for button and code cleanup
- move style for social button to step page
- ham/eng 478 create poap check (#540)
- itsmekori/eng-637-update-saga-packpuzzles-in-redwood (#638)
- itsmekori/eng-639-redwood-update-lens-collector (#640)
- migrate nft metadata (#641)
- Itsmekori/eng 646 add to redwood (#643)
- bug/node fetch (#645)
- itsmekori/eng-656-remove-duplicate-link (#648)
- ENG-459: implement magiclink auth (#598)
- ENG-554: Steps standalone cell (#557)
- ENG-626: save user attempts to db (#631)
- bug/env (#659)
- add packs to seed.js (#646)
- ENG-679: use sets for wrappers (#660)
- ENG-680: link nft to rewardable (#661)
- ENG-671: roles for user data (#655)
- test pr (#668)
- ENG-603: Claim page and minting (#619)
- ENG-704: little fixes (#672)
- ENG-708: add-gnosis-chain-to-rw (#674)
- use profile dropdown instead of button. update mobile sizings
- add migrate to build command
- fix auth
- small profile cell tweak
